### ISSUE TITLE
french custom game settings not using required special character (#290)

### ISSUE THREAD

**Issue by yubber** [Zezombye/overpy#310]
the bug got out of the bag again (originally issue #290)

game expects ` ` (U+00a0) when specifying a team (`Équipe 1`), instead of the normal space

this character is missing in the "max players in team 1/2" and "heroes" settings
```
salon
	{
		Maximum de joueurs en Chacun pour soi: 4
		Joueurs max. dans Équipe 1.: 2 <-- here
		Joueurs max. dans Équipe 2.: 2 <-- here
	}
...
héros
	{
		Équipe 1 <-- here, and also in Équipe 2
		{
			héros activés
			{
				Ange
			}
		}
...
```

**Comment by Zezombye** [comment]
Yes, this is a datatool bug (they strip them out because it caused issues on other stuff)

What I could do is merge an old version of the strings file with the current one, and mark the guids with a space to not update, but for now I don't have much time to do this, you'll have to switch to English.

**Comment by yubber** [comment]
i found another, if/when you're free to fix it, the same character is expected in:
- cooldown settings (previously `Temps de recharge de X` -> `Temps de recharge_: X)
- soldier's name (Soldat_: 76)

where _ is U+00a0

i am going to throw a big rock at blizzard headquarters
